deploy/base/toolforge-buildpacks-phases-patch.json

```diff
@@ -16,7 +16,7 @@
   },
   {
```

Comment:
This file needs a new patch to replace the bash pod registry for the new step.

Reply:
True

Reply:
I'd love to move this to helm xd